sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -101,7 +101,29 @@ add_sycl_plugin(level_zero
     "../unified_runtime/ur/usm_allocator_config.cpp"
     "../unified_runtime/ur/usm_allocator_config.hpp"
     "../unified_runtime/ur/adapters/level_zero/ur_level_zero.hpp"
+    "../unified_runtime/ur/adapters/level_zero/ur_level_zero_common.hpp"
```

Comment:
looks you are missing "kernel" here, is that intentional?

Reply:
right, kernel code would go in the module file, at least for now.